packages/server/billing/helpers/teamLimitsCheck.ts

```diff
@@ -35,7 +35,7 @@ const enableUsageStats = async (userIds: string[], orgId: string) => {
     .table('OrganizationUser')
     .getAll(r.args(userIds), {index: 'userId'})
     .filter({orgId})
-    .update({suggestedTier: 'team'})
+    .update({suggestedTier: 'starter'})
```

Comment:
The org has exceeded the starter tier limits, so we call `enableUsageStats`. I'm not sure why the `suggestedTier` would be `starter` rather than `team` as we're suggesting that they upgrade?

Reply:
@nickoferrall I see a couple of issues with specifying `team` as a `suggestedTier` :

1. We have hardcoded logic that shows a nagging snackbar in this case, which we don't want: https://github.com/ParabolInc/parabol/blob/master/packages/client/hooks/useUsageSnackNag.ts#L22
2. As a workaround I checked if there is an org with `limitExceededAt` flag. It works, but not perfectly:
3. If the user fixed its limit by removing a team, he will see nagging snackbar again, unless we disable usage stats for him.
4. But we can't just disable usage stats, we also need to remove feature flag. Because usage stats does not work without `suggestedTier`.

I think we just want to enable usage stats without suggesting anything, so the above approach feels a bit hacky.

Setting `personal` solves these issues in a quick way without much refactoring.

What do you think?

Reply:
I think it makes sense as a quick solution that helps us to ship the feature faster, but I think we should create an issue to refactor it soon

Reply:
Agree, I can create as soon as I get back

Reply:
-2 Ok, I think I get now what you're trying to achieve. I still believe setting `suggestedTier: 'starter'` when they run over the limit is wrong, we should still set `suggestedTier: 'team'` here. However we can change `maybeRemoveRestrictions` to set the `suggestedTier: 'starter'` if they fall below the limit again. This should still accomplish what you wanted to do, shouldn't it?

Reply:
@Dschoordsch this would work, updated the description and pushed new changes